scripts/deploy_prod.sh

```diff
@@ -8,26 +8,15 @@ if ! git diff-index --quiet HEAD; then
   exit
 fi
 
-# Merge master into master_clever, then push master_clever
-# Deployment to Clever Cloud is actually triggered via a hook
-# on a push on this branch
-
-# synchronize master and master_clever by replaying the local branches on top of remote ones
+# Synchronize `master` and `master_clever` by replaying the local branches on top of remote ones
 git fetch origin
-git checkout master
-git rebase origin/master master
-git checkout master_clever
-# Since we changed the deployment process, a hard reset is needed to resync local master_clever with origin
-git reset --hard origin/master_clever
-
-# merge master into master_clever
-git merge master --no-edit --ff-only
-git push origin master_clever
+git rebase origin/master master  # Update the local `master` with the remote one, we will be on `master` after that
+git push origin master:master_clever  # A deployment is triggered on Clever Cloud when pushing `master_clever`
```

Comment:
En lisant cela, j'ai l'impression que si quelqu'un a un commit qui traine sur son master à lui il peut par inadvertance le pousser directement sur origin/master_clever sans l'avoir jamais poussé sur origin/master non ? (et oui c'était déjà le cas avant, mais ça serait pas mal de changer ça au passage, dans un second commit :) )

Reply:
Je me demande si ceci ne fonctionnerais pas en fait :thinking: 
```suggestion
git push origin origin/master:master_clever  # A deployment is triggered on Clever Cloud when pushing `master_clever`
```
Sinon peut-être avec `git push refs/remotes/origin/master:master_clever`